src/review_gator/tox_runner.py

```diff
@@ -12,20 +12,33 @@ def prep_tox_state(output_directory=None, mp_id=None):
     abs_vendor_path = os.path.join(os.path.dirname(
         os.path.realpath(__file__)), "vendor")
     tox_state = os.path.join(output_directory, "{}.svg".format(mp_id))
+    tox_output = os.path.join(output_directory, "tox-{}.output.txt"
+                              .format(mp_id))
     clock_svg = os.path.join(abs_vendor_path, "clock.svg")
+    tox_output_dummy = os.path.join(abs_vendor_path, "tox-output.txt")
     shutil.copy(clock_svg, tox_state)
+    shutil.copy(tox_output_dummy, tox_output)
 
 
-def run_tox(source_repo, source_branch, output_directory=None, mp_id=None):
+def run_tox(source_repo, source_branch, output_directory=None, mp_id=None,
+            parallel_tox=True):
     abs_vendor_path = os.path.join(os.path.dirname(
         os.path.realpath(__file__)), "vendor")
     tox_state = os.path.join(output_directory, "{}.svg".format(mp_id))
+    tox_output = os.path.join(output_directory, "tox-{}.output.txt"
+                              .format(mp_id))
     clock_svg = os.path.join(abs_vendor_path, "clock.svg")
     error_svg = os.path.join(abs_vendor_path, "error.svg")
     success_svg = os.path.join(abs_vendor_path, "success.svg")
     shutil.copy(clock_svg, tox_state)
+
     try:
-        tox_return_code = lpmptox_runtox(source_repo, source_branch)
+        tox_return_code = lpmptox_runtox(
+            source_repo,
+            source_branch,
+            tox_command='tox --recreate --parallel auto'
```

Comment:
With parallel invocation we get what should be a spinning indicator for pages of output like:
```
[K
â ¼ [1] py27
[K
â ´ [1] py27
[K
â ¦ [1] py27
```

When parallel_tox is true, please consider adding --quiet.  I've compared the output locally with/without --quiet and a little is lost (list of packages/versions installed, PYTHONHHASHSEED, and command) but it cleans up the output nicely so that we can find errors in a log file.

Reply:
Nice suggestion. Will test

Reply:
@rcj Change to `-q` added